companion/src/firmwares/er9x/er9xinterface.cpp

```diff
@@ -91,7 +94,9 @@ inline void applyStickModeToModel(Er9xModelData & model, unsigned int mode)
 
 bool Er9xInterface::loadxml(RadioData &radioData, QDomDocument &doc)
 {
+  #ifdef DEBUG
```

Comment:
because I thought there where no QT libs on those files... just tested and it works, I was mistaken... :wink: